src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -844,11 +844,13 @@ public function cursorPaginate($perPage = null, $columns = ['*'], $cursorName =
 
         $parameters = $orders->pluck('column')->toArray();
 
+        $originals = $this->reverseColumnAliasingForCursorPagination($parameters);
+
         if (! is_null($cursor)) {
             if (count($parameters) === 1) {
-                $this->where($column = $parameters[0], $comparisonOperator, $cursor->parameter($column));
+                $this->where($originals[0], $comparisonOperator, $cursor->parameter($parameters[0]));
```

Comment:
`$this->where(array_shift($originals), $comparisonOperator, $cursor->parameter(array_shift($parameters)));`